cmd/relaynode/README.md

```diff
@@ -0,0 +1,35 @@
+# Relay Node For Qitmeer Network
+
+Qitmeer Relay node is a kind of node which uses circuit transport protocol. 
+It can route communication between two peers. So it can also be used to help 
+private network qitmeer nodes to "Hole-punching".
+
+
+Relay node connections are end-to-end encrypted, which means that the peer acting as the relay is unable to read or tamper with any traffic that flows through the connection.
+
+
+## Installation
+```bash
+~ cd ./cmd/relaynode
+~ go build
+~ ./relaynode -h
+```
+## Getting Started
+
+* Start `relaynode`
+```bash
+~ ./relaynode
```

Comment:
```sh
tony@dagfans ~/github.com/qitmeer/qitmeer/cmd/relaynode (master-p2p*) $ ./relaynode
2020-11-17|11:09:22.011 [INFO ] Start relay node...                 module=RELAY
2020-11-17|11:09:22.062 [INFO ] Load config completed               module=RELAY
2020-11-17|11:09:22.062 [INFO ] NetWork:mixnet  Genesis:119dbc54464156d342c6bc4655d10c86ce9be002cd8eac9bd83f5d2bbe3e0b4c module=RELAY
2020-11-17|11:09:22.062 [INFO ] Run relay node...                   module=RELAY
2020-11-17|11:09:22.071 [ERROR] Failed to create host %v            module=RELAY LOG15_ERROR= LOG15_ERROR="Normalized odd number of arguments by adding nil"
2020-11-17|11:09:22.076 [INFO ] Stop relay node                     module=RELAY
2020-11-17|11:09:22.076 [ERROR] failed to listen on any addresses: [listen tcp4 0.0.0.0:1001: bind: permission denied listen tcp4 103.1.154.227:1001: bind: permission denied] module=RELAY
```

Reply:
sudo is OK

```sh
tony@dagfans ~/github.com/qitmeer/qitmeer/cmd/relaynode (master-p2p*) $ sudo ./relaynode
2020-11-17|11:17:40.701 [INFO ] Start relay node...                 module=RELAY
2020-11-17|11:17:40.743 [INFO ] Load config completed               module=RELAY
2020-11-17|11:17:40.744 [INFO ] NetWork:mixnet  Genesis:119dbc54464156d342c6bc4655d10c86ce9be002cd8eac9bd83f5d2bbe3e0b4c module=RELAY
2020-11-17|11:17:40.744 [INFO ] Run relay node...                   module=RELAY
2020-11-17|11:17:40.751 [INFO ] Relay Address: /ip4/103.1.154.227/tcp/1001/p2p/16Uiu2HAmA1vRXRU4witxhXouDtLQdCCNfHaJXcQFG9jrtr8aXsXv
 module=RELAY
2020-11-17|11:17:40.751 [INFO ] You can copy the relay address and configure it to the required Qitmeer-Node module=RELAY
```

Reply:
The default port 1001 is within the system service reserved range (1~1024), please change it greater to work around  the sudo requirement

Reply:
https://github.com/Qitmeer/qitmeer/pull/396/commits/c748f021a14035e3c5ce39ee19375ed3c1c53017

Reply:
> c748f02

verified